Fubernetes.UseLocal/Program.fs

```diff
@@ -1,123 +1,144 @@
-﻿open Fubernetes.Configuration
+﻿open Fubernetes.Service
+open Fubernetes.Secret
+open Fubernetes.Configuration
 open Fubernetes.Resources
 
 let createSimpleSecret () =
     let clusterIpService =
-        new Service.ClusterIPService(
-            { Name = "svc-clusterip-01"
-              Namespace = "default"
-              Selector = [ ("app", "nginx"); ("network", "main-svc") ]
-              Ports =
-                [ { Name = "tcp-port"
-                    Protocol = ServiceShared.KubernetesProtocol.TCP
-                    Port = 8000
-                    TargetPort = 8080 }
-                  { Name = "udp-port"
-                    Protocol = ServiceShared.KubernetesProtocol.UDP
-                    Port = 8001
-                    TargetPort = 8081 } ] }
+        new ClusterIP(
+            { ClusterIPConstructor.Default with
+                Metadata =
+                    { Name = "svc-clusterip-01"
+                      Namespace = "default" }
+                Spec =
+                    { Selector = ([ "app", "nginx"; "network", "main-svc" ] |> Map.ofList)
+                      Ports =
+                        [ { Name = "tcp-port"
+                            Protocol = ServiceProtocol.TCP
+                            Port = 8000
+                            TargetPort = 8080 }
+                          { Name = "udp-port"
+                            Protocol = ServiceProtocol.UDP
+                            Port = 8001
+                            TargetPort = 8081 } ]
+                      Type = ServiceKind.ClusterIP } }
```

Comment:
This is pretty ugly in my opinion. This is not the experience I'd like to have when creating my manifests.